server/src/main/java/io/crate/planner/operators/EquiJoinDetector.java

```diff
@@ -51,15 +51,10 @@ public class EquiJoinDetector {
 
     private static final Visitor VISITOR = new Visitor();
 
-    public static boolean isHashJoinPossible(JoinType joinType, Symbol joinCondition) {
-        if (joinType != JoinType.INNER) {
+    public static boolean isEquiJoinCondition(Symbol joinCondition) {
```

Comment:
```suggestion
    public static boolean isEquiJoin(Symbol joinCondition) {
```

The `condition` part is captured by the parameter name.

Reply:
👍